src/main/java/org/generationcp/middleware/hibernate/DataSourceProperties.java

```diff
@@ -73,10 +73,20 @@
 
 	public DataSourceProperties(final Properties properties) {
 
-		this.host = this.getPropertyValue(properties, this.host, DataSourceProperties.DB_HOST);
-		this.port = this.getPropertyValue(properties, this.port, DataSourceProperties.DB_PORT);
-		this.userName = this.getPropertyValue(properties, this.userName, DataSourceProperties.DB_USERNAME);
-		this.password = this.getPropertyValue(properties, this.password, DataSourceProperties.DB_PASSWORD);
+		this.host = System.getenv("BMS_DB_HOST");
+
+		if(this.host != null) {
+			this.host = System.getenv("BMS_DB_HOST");
```

Comment:
I think it's safer to not assume that if BMS_DB_HOST system environment is present, that the others would be present as well